ibm/resource_ibm_database.go

```diff
@@ -147,24 +142,54 @@ func resourceIBMDatabaseInstance() *schema.Resource {
 				ForceNew:    true,
 			},
 			"members_memory_allocation_mb": {
-				Description: "Memory allocation required for cluster",
-				Type:        schema.TypeInt,
-				Optional:    true,
-				Computed:    true,
-				//ValidateFunc: validation.IntBetween(2048, 114688),
+				Description:   "Memory allocation required for cluster",
+				Type:          schema.TypeInt,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"node_count", "node_memory_allocation_mb", "node_disk_allocation_mb", "node_cpu_allocation_count"},
```

Comment:
So there is lots of repeating code in here. Is there a way that we can do something like factor these things out and reference them (or, like for the code below) us functions?